### PR TITLE
nsxiv: update to 28

### PR DESCRIPTION
**Pull request by WitherCubes**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by WitherCubes** [comment]
@ericonr Sorry for closing the original PR and opening a new one, the whole repo got messed up showing errors. I have done all the changes you told to do.

File: srcpkgs/nsxiv/template [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # Template file for 'nsxiv'
 pkgname=nsxiv
-version=27.1
+version=28
 revision=1
 build_style=gnu-makefile
```
Comment (N-R-K) [inline srcpkgs/nsxiv/template:5]: I'm not familiar with void, but nsxiv v28 doesn't use any non-posix/gnu extension in the Makefile. If there is some non posix extension being used, then it should be a bug report on nsxiv repo.

File: srcpkgs/nsxiv/template [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
 CFLAGS="-I${XBPS_CROSS_BASE}/usr/include/freetype2"
 
 pre_build() {
 	[ -e ${FILESDIR}/config.h ] && cp ${FILESDIR}/config.h config.h ||:
 }
 
-post_install() {
-	vinstall nsxiv.desktop 644 usr/share/applications
-}
+make_install_target=install-all
```
Comment (paper42) [inline srcpkgs/nsxiv/template:20]: Move this line to the main block.

File: srcpkgs/nsxiv/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/nsxiv/template:5]: No, this is correct.

**Pull request merged**